angr/analyses/cfg_accurate.py

```diff
@@ -990,7 +990,8 @@ def _process_hints(self, analyzed_addrs, remaining_entries):
                                           new_state,
                                           self._context_sensitivity_level
                                           )
-                remaining_entries.append(new_path_wrapper)
+                self._insert_entry(new_path_wrapper)
```

Comment:
Oh interesting. It might be an issue with constant refactor and code merge. I'll look at it.

EDIT: Yes, I think you are right, I'm going to merge in this PR. Thanks!